src/server/mail.ts

```diff
@@ -33,3 +33,33 @@ export const mailgunMessage = async(
     }
   }
 };
+
+export const mailgunRegister
+ = async(email: string, username: string, password: string): Promise<void> => {
+   try {
+     const data = {
+       from: 'trevinhofmann@gmail.com',
+       to: ['jraugustyn07@gmail.com'], // set email here
+       subject: 'Your Sunny Software Regisation:',
+       text: `An account has been created for you at Sunny Software. You may log in with the following credentials:
+
+      Username: ${username}
+      Password: ${password}
+
+      You may change your password after login.
```

Comment:
I'll work on the text content for this and everything else later, but this should probably be "Please change your password after login."